### PR TITLE
domainname-fix

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

File: actions/domainname-change [hunk 1]
```diff
@@ -19,9 +19,18 @@
 domainname="$1"
 hostname=$(hostname)
 
-if grep -q 127.0.1.1 /etc/hosts ; then
-    sed -i "s/127.0.1.1.*/127.0.1.1       $hostname.$domainname $hostname/" /etc/hosts
+if domainname ; then
```
Comment (jvalleroy) [inline actions/domainname-change]: Is this meant to run the domainname command?

**Comment by mridulnagpal** [comment]
It is just to check that if domain name is NULL then it won't print the
hostname. in the second line of etc/hosts file. Therefore I put up the if
condition to check what the user entered. This should work according to me.

**Comment by jvalleroy** [comment]
It looks like domainname always has a zero (success) exit code, even when the domain name is not set.
```
vagrant@freedombox:~$ domainname && echo yes || echo no
(none)
yes
```
So the if condition will always be "true".
```
vagrant@freedombox:~$ if domainname ; then echo yes; else echo no; fi
(none)
yes
```

**Comment by mridulnagpal** [comment]
I made the changes now it seems to work fine..

**Comment by jvalleroy** [comment]
Merged with a few fixes:
- Use -z to check if domainname is empty.
- Add space before ].
- Remove extra spaces.
- Add newline at end of file.

**Pull request closed without merging**